Scripts/script-IncapScheduleTask.yml

```diff
@@ -0,0 +1,39 @@
+commonfields:
```

Comment:
Is there a description to this script? It does not look like a generic script but for some purpose...